src/styles/Colors.js

```diff
@@ -1,7 +1,7 @@
 export default Colors = {
-  primary: '#F4F4F4',
-  secondary: '#FFFFFF',
-  highlight: '#FF5553',
+  primary: '#89bdd3',
```

Comment:
I like that idea also i think im graphically impaired a little so ill trust you haha